features/netsocket/NetworkInterface.h

```diff
@@ -253,6 +253,18 @@ class NetworkInterface: public DNS {
      */
     virtual nsapi_error_t gethostbyname_async_cancel(int id);
 
+    /** Get a domain name server from a list of servers to query
+     *
+     *  Returns a DNS server address for a index. If returns error no more
```

Comment:
Nit-picking, but the English here could do a bit of fine-tuning. I assume we store the server address to the @param address and also it's name to @param interface_name AND NS_API_ERROR_OK, if a DNS server is availble in the index queried location? If no DNS server in index queries location, we return some negative error code, right?

Reply:
Member get_dns_server  is removed.